internal/provider/resource_cloudflare_worker_script_test.go

```diff
@@ -51,7 +51,7 @@ func TestAccCloudflareWorkerScript_MultiScriptEnt(t *testing.T) {
 			{
 				Config: testAccCheckCloudflareWorkerScriptConfigMultiScriptUpdateBinding(rnd),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckCloudflareWorkerScriptExists(name, &script, []string{"MY_KV_NAMESPACE", "MY_PLAIN_TEXT", "MY_SECRET_TEXT", "MY_WASM"}),
+					testAccCheckCloudflareWorkerScriptExists(name, &script, []string{"MY_KV_NAMESPACE", "MY_PLAIN_TEXT", "MY_SECRET_TEXT", "MY_WASM", "MY_SERVICE_BINDING"}),
```

Comment:
looks like this is failing because the test is asserting the presence against the API response however, it's not there.

```
TF_ACC=1 go test $(go list ./...) -v -run "^TestAccCloudflareWorkerScript_" -count 1 -parallel 1 -timeout 120m -parallel 1
?       github.com/cloudflare/terraform-provider-cloudflare     [no test files]
=== RUN   TestAccCloudflareWorkerScript_Import
--- PASS: TestAccCloudflareWorkerScript_Import (13.64s)
=== RUN   TestAccCloudflareWorkerScript_MultiScriptEnt
=== PAUSE TestAccCloudflareWorkerScript_MultiScriptEnt
=== CONT  TestAccCloudflareWorkerScript_MultiScriptEnt
    resource_cloudflare_worker_script_test.go:27: Step 3/3 error: Check failed: Check 1/3 error: cannot find binding with name MY_SERVICE_BINDING
--- FAIL: TestAccCloudflareWorkerScript_MultiScriptEnt (22.85s)
FAIL
FAIL    github.com/cloudflare/terraform-provider-cloudflare/internal/provider   37.110s
?       github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/changelog-check   [no test files]
?       github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/maintainer-only-file-check        [no test files]
?       github.com/cloudflare/terraform-provider-cloudflare/tools/cmd/tf-log-check      [no test files]
?       github.com/cloudflare/terraform-provider-cloudflare/version     [no test files]
FAIL
make: *** [testacc] Error 1
```

here is an example HTTP request payload from the tests that shows it isn't being created

```
PUT /client/v4/accounts/f037e56e89293a057740de681ac9abbe/workers/scripts/aqimbzllgz HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform/1.1.9 terraform-plugin-sdk/2.10.1 terraform-provider-cloudflare/dev
Content-Length: 984
Content-Type: multipart/form-data; boundary=4d65ebd5270196f82a97294f26ee0de30c5b358ee5140f49c9474923f81d
X-Auth-Email: [redacted]
X-Auth-Key: [redacted]
Accept-Encoding: gzip
Connection: close

--4d65ebd5270196f82a97294f26ee0de30c5b358ee5140f49c9474923f81d
Content-Disposition: form-data; name="metadata"
Content-Type: application/json

{"body_part":"script","bindings":[{"name":"MY_KV_NAMESPACE","namespace_id":"f43bc84c7cb94410b931f49e6124a8fe","type":"kv_namespace"},{"name":"MY_PLAIN_TEXT","text":"aqimbzllgz","type":"plain_text"},{"name":"MY_SECRET_TEXT","text":"aqimbzllgz","type":"secret_text"},{"name":"MY_WASM","part":"03d0a98d438c7150e36afc0ce514a596","type":"wasm_module"}]}
--4d65ebd5270196f82a97294f26ee0de30c5b358ee5140f49c9474923f81d
Content-Disposition: form-data; name="script"
Content-Type: application/javascript

addEventListener('fetch', event => {event.respondWith(new Response('test 2'))});
--4d65ebd5270196f82a97294f26ee0de30c5b358ee5140f49c9474923f81d
Content-Disposition: form-data; name="03d0a98d438c7150e36afc0ce514a596"
Content-Type: application/wasm

 asm   ����  
--4d65ebd5270196f82a97294f26ee0de30c5b358ee5140f49c9474923f81d--
```

Reply:
ah, think i found it. missing from https://cs.github.com/cloudflare/terraform-provider-cloudflare/blob/4e4d26b4ebd741e58ad5075cace549745f044352/internal/provider/resource_cloudflare_worker_script.go?q=parseWorker#L67-L96

Reply:
Oh I see the issue, I forgot to handle service bindings in parseWorkerBindings.

Reply:
What are you running to get these test results? I'm not seeing these failures in the GitHub CI

Reply:
`make testacc` doesn't run on PRs as it creates/destroys real resources (see https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/development.md); i manually run it on PRs and it automatically runs on master merges.